components/UI/textfield/GoogleMapsAuthComplete.tsx

```diff
@@ -0,0 +1,189 @@
+import * as React from "react"
+import Box from "@mui/material/Box"
+import TextField from "@mui/material/TextField"
+import Autocomplete from "@mui/material/Autocomplete"
+import LocationOnIcon from "@mui/icons-material/LocationOn"
+import Grid from "@mui/material/Grid"
+import Typography from "@mui/material/Typography"
```

Comment:
I prefer import {Box, TextField,...} from '@mui/material' but it's fine

Reply:
I think it's not yet done and some parts look like they are not from you 🤔

Reply:
I copied this whole file form MUI and make some adjustments.